plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -174,10 +174,16 @@
      * The OpenShift deploy timeout in seconds:
      * See this issue for background of why for end users on slow wifi on their laptops
      * DeploymentConfigs usually barf: https://github.com/openshift/origin/issues/10531
+     *
+     * Lets default to 2 hours which should be plenty of time on a slow connection to download
+     * all the images for a YAML/JSON manifest (which can contain many separate images)
      */
-    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds")
+    @Parameter(property = "fabric8.openshift.deployTimeoutSeconds", defaultValue = "7200")
```

Comment:
I a short timeout can help developers when they are developing the app; if you know the image is local you may wanna go shorter - to 30 seconds or something. Its very easy to enable this setting in your `~/.m2/settings.xml`

The issue with the default OOTB is when you make an app as an openshift YAML / Template it means you've no idea if the end user can ever even run your app without manually retrying deployments as it totally depends on their network speed - which is a kinda crappy UX if you're trying to make reusable apps for end users on openshift. Especially if your app, like fabric8 itself, has like 7-10 DCs inside which are likely to fail for most folks who don't have crazy fast wifi.